Firestore/core/src/local/leveldb_key.h

```diff
@@ -635,6 +643,76 @@ class LevelDbRemoteDocumentReadTimeKey {
   model::SnapshotVersion read_time_;
 };
 
+/**
+ * A key in the bundles table, storing the bundle Id for each entry.
+ */
+class LevelDbBundlesKey {
```

Comment:
Even though the table is plural ("bundles"), each individual entry in it is singular ("bundle"). This should be a `LevelDbBundleKey` because the key addresses a specific bundle.

Reply:
Done.